### ISSUE TITLE
Reload ToDo file

### ISSUE THREAD

**Issue by sttrebo** [QTodoTxt/QTodoTxt#379]
reference:  Closing when deleting todo file #328 
Can I ask for an alternative soultion?  I use IFTTT to add lines to my todo.txt file using several IFTTT applets.  When those applets fire, the way they seem to work, is that the todo file is deleted and a new file is written.  works fine with Qtodo seeing this and you get the dialog that the file is missing (per the issue #328 ).  Any way that I can get an option to just re-load the file instead of the current dialog?

What happens now is when the IFTTT applet fires and the new todo.txt file is written, Qtodo see's that and the current dialog  pops up.  good, except if you just hit 'ok' without re-starting Qtodo then any changes you make in Qtodo will overwrite the changes in the todo.txt file so the IFTTT applet changes get lost.  a 'reload' button would get around this use problem.

**Comment by oroulet** [comment]
I agree that users should be able to choose between reload or overwrite.
Should be easy. Do you try?

**Comment by blackchr** [comment]
My use case is to update my todo.txt file on a PC and a phone.  Not simultaneously, but interleaved (with reliable syncing, so no conflicts).  This almost always causes QTodoTxt to show a dialog about the missing file, and I then have to reopen the file through the menus.

My experience would be greatly improved if QTodoTxt would attempt to reopen the file automatically when the file changes (or disappears and reappears), as long as there are no local changes which haven't been written to disk yet.

**Comment by oroulet** [comment]
will have a look. We should reload if no changes in current model

**Comment by oroulet** [comment]
fixed in QTodoTxt2 but I am not sure I want to port that that heavy cleanup to version 1... https://github.com/QTodoTxt/QTodoTxt2/commit/3ccc45500b52aee1056259e58a087b03c7d6690e
Not sure I would advice to use version 2 either. but soon...
